scan_line_sum: 2161
scan_angle: 270
referenced_laser_num: 50
offset_angle_camera_to_lidar: 0

detected_image_topic_name: /mask_rcnn/detected_image
masks_topic_name: /mask_rcnn/masks
# scan_topic_name: /scan
scan_topic_name: /disturbing_points_detector/filtered_laser
